diff --git a/libtcc.h b/libtcc.h
index e69cc6b..e5bd314 100644
--- a/libtcc.h
+++ b/libtcc.h
@@ -90,6 +90,9 @@ LIBTCCAPI int tcc_relocate(TCCState *s1, void *ptr);
    returns -1 if error. */
 #define TCC_RELOCATE_AUTO (void*)1
 
+/* Same as `tcc_relocate(s1, TCC_RELOCATE_AUTO)` */
+LIBTCCAPI int tcc_relocate_auto(TCCState *s1);
+
 /* return symbol value or NULL if not found */
 LIBTCCAPI void *tcc_get_symbol(TCCState *s, const char *name);
 
diff --git a/tccrun.c b/tccrun.c
index 50178a8..c115a7c 100644
--- a/tccrun.c
+++ b/tccrun.c
@@ -88,6 +88,11 @@ LIBTCCAPI int tcc_relocate(TCCState *s1, void *ptr)
     return ret;
 }
 
+/* Same as `tcc_relocate(s1, TCC_RELOCATE_AUTO)` */
+LIBTCCAPI int tcc_relocate_auto(TCCState *s1) {
+    return tcc_relocate(s1, TCC_RELOCATE_AUTO);
+}
+
 /* launch the compiled program with the given arguments */
 LIBTCCAPI int tcc_run(TCCState *s1, int argc, char **argv)
 {
